volumina/widgets/dataExportOptionsDlg.py

```diff
@@ -410,7 +408,8 @@ def __init__(self, parent, inputSlot):
 
         def validate(self, userAxes, pos):
             taggedShape = self._inputSlot.meta.getTaggedShape()
-            inputAxes = list(taggedShape.keys())
+            originalAxes = self._inputSlot.meta.getOriginalAxisKeys()
```

Comment:
shouldn't we just allow users to create all the axes from `{t, c, z, y, x}` they want? @Tomaz-Vieira

Reply:
okay I went ahead to allow all of those when specifying output. There is not reason users shouldn't be able to export 5D if they choose to.